### ISSUE TITLE
Inconsistency of results on MDD task

### ISSUE THREAD

**Issue by Jason3900** [Alue-Benchmark/alue_baselines#6]
Hey, 
I found that the submission results of MDD task is not the same as the ones I run locally. I'm wondering how does that happen.

**Comment by hseelawi** [comment]
Hello,

We use f1 score with macro weighing. The implementation we use is that of
sklearn.metrics.f1_score. Would you please let us know what f1 score
implementation are you using?

**Comment by Jason3900** [comment]
Yeah, I use the exact same metric but it turns out to have 1pp gap. However, other f1-macro task such as fid doesn't seem to have this issue.

**Comment by hseelawi** [comment]
Would you please try using `scikit-learn 1.0.1`, just to rule out any dependency related issues? I have also resent you the dataset splits.

**Comment by hseelawi** [comment]
@Jason3900 given our corrospondence via email, I believe this issue is solved now. I am gonna close it, but please feel free to open it otherwise.

**Comment by Jason3900** [comment]
Thanks a lot!
